scripts/publish.sh

```diff
@@ -302,6 +327,8 @@ sync_latest_image() {
 	fi
 	create_manifest_list ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit "stable" ${AWS_FOR_FLUENT_BIT_STABLE_VERSION} || echo "0"
 
+	create_manifest_list_init ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit "init-stable" ${AWS_FOR_FLUENT_BIT_STABLE_VERSION} || echo "0"
```

Comment:
This code has changed: https://github.com/aws/aws-for-fluent-bit/pull/571#discussion_r1123894361